src/notify/notify.c

```diff
@@ -188,7 +206,9 @@ static int parse_argv(int argc, char *argv[]) {
         assert(argc >= 0);
         assert(argv);
 
-        while ((c = getopt_long(argc, argv, "h", options, NULL)) >= 0) {
+        while ((c = getopt_long(argc, argv,
+                                (arg_action == ACTION_FORK || do_exec) ? "+hq" : "hq",
```

Comment:
Are you sure this works at all? If I read https://github.com/bminor/glibc/blob/master/posix/getopt.c#L482-L485 correctly, the posixly_correct is only initialized once for the first getopt_long() invocation.